### PR TITLE
Remove double call to config when first run

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Signed-off-by: Pierre-Emmanuel Jacquier <pierre-emmanuel.jacquier@epitech.eu>

### CODE REVIEW

**Comment by pierre-emmanuelJ** [comment]
Problem:

If no config file exist `exo` try to build config file by calling `generateConfig()`

so if you call `exo config` if no config file exist it will call config two time.

**Comment by greut** [comment]
how can I reproduce the issue?

**Comment by pierre-emmanuelJ** [comment]
Not on this commit:

Remove all your config file, then `$ exo config`

**Comment by greut** [comment]
`config` needs a lot of love.

File: cmd/exo/cmd/root.go [hunk 1]
```diff
@@ -104,15 +105,56 @@ func initConfig() {
...
-			log.Fatal(err)
-		}
-		configFilePath = path
+	if cfgFilePath != "" {
+		configFilePath = cfgFilePath
+		return
+	}
+
+	if getCmdPosition("config") == 1 {
+		ignoreBuilClient = true
```
Comment (greut) [inline cmd/exo/cmd/root.go]: typo?

Comment (pierre-emmanuelJ) [reply cmd/exo/cmd/root.go]: `ignoreClientBuild` instead of `ignoreBuilClient` and missing `d` to build ! thanks 👍

**Pull request merged**